Content.Shared/Nuke/NukeDiskComponent.cs

```diff
@@ -1,9 +1,28 @@
 using Robust.Shared.GameStates;
+using Robust.Shared.Map;
 
 namespace Content.Shared.Nuke;
 
 /// <summary>
-/// Used for tracking the nuke disk - isn't a tag for pinpointer purposes.
+/// Used for tracking and respawning the nuke disk - isn't a tag for pinpointer purposes.
 /// </summary>
 [RegisterComponent, NetworkedComponent]
-public sealed class NukeDiskComponent : Component {}
+public sealed class NukeDiskComponent : Component
+{
+    [ViewVariables]
+    [DataField("station")]
+    public EntityUid Station = EntityUid.Invalid;
+
+    [ViewVariables]
+    [DataField("stationMap")]
+    public (MapId?, EntityUid?) StationMap;
```

Comment:
Maybe this should keep track of the nuke instead?

Reply:
Maybe? Eventually it should probably be tracked so the disk itself doesn't go off the map.